detekt.yml

```diff
@@ -2,3 +2,6 @@ style:
   VarCouldBeVal:
     ignoreLateinitVar: true
     includes: [ "**/test/**" ]
+formatting:
+  Indentation:
+    active: false
```

Comment:
don't disable it completely, but configure options

Reply:
What do you mean by options?

Reply:
<img width="423" alt="image" src="https://github.com/user-attachments/assets/9eed5aa6-2dde-4e9f-8730-0fa055463f37">
If I understand correctly, indentSize in IDEA and detekt are different and you can align them